UnityProject/Assets/Scripts/Clothing/WearableArmor.cs

```diff
@@ -18,7 +18,7 @@ public class WearableArmor : MonoBehaviour, IServerInventoryMove
 	[Tooltip("What body parts does this item protect")]
 	private BodyPartsCovered bodyPartsCovered = BodyPartsCovered.None;
 
-	private PlayerHealth player;
+	private PlayerHealthV2 player;
```

Comment:
Codacy found an issue: [Remove the field 'player' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)